catalog-rest-service/src/main/resources/ui/src/components/SchemaTab/SchemaTab.component.tsx

```diff
@@ -128,7 +128,7 @@ const SchemaTab: FunctionComponent<Props> = ({
       </div>
       <div className="row">
         <div className="col-sm-12">
-          {checkedValue === 'schema' ? (
+          {checkedValue === 'schema' && columns?.length ? (
```

Comment:
How will it look if there are no columns?

Reply:
And not sample-data as well

Reply:
If there are no columns then it won't render the EntityTable.

Reply:
> And not sample-data as well

if there is no sample data then it won't render the sample data table.